config/locales/en.yml

```diff
@@ -4,7 +4,7 @@ en:
     sessions:
       create:
         session_expired: 'Your session has expired, please sign in again.'
-        email_sent_if_record_found: "If we found you in the system, we've sent you an email."
+        email_sent_if_record_found: "If we found you in the system, we have sent you an email."
```

Comment:
I changed wording since rails actionview escapes the rendered content of `we've` into some html entities which were hard to capture in the integration test.

Another solution would be to stub the I18n.translate-call or change the backend/translations just for the test.

Open for discuission here.

Reply:
Wondered why you'd changed this. Reason sounds like a real hairball. Fine with this or third option below.